packages/workflow/tsconfig.test.json

```diff
@@ -0,0 +1,7 @@
+{
+	"extends": "./tsconfig.json",
+	"include": [
+		"test/**/*.d.ts",
+		"test/**/*"
+	]
+}
```

Comment:
Why is this needed?

Reply:
Our `tsconfig.json` only includes `src/**/*.ts`. This will add typescript coverage to our `test` folder without adding it to the build output.